admob-latest/src/test/java/com/rakuten/tech/mobile/admob/AdMobDisplayer20Spec.kt

```diff
@@ -0,0 +1,112 @@
+package com.rakuten.tech.mobile.admob
+
+import android.app.Activity
+import androidx.test.core.app.ActivityScenario
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.android.gms.ads.interstitial.InterstitialAd
+import com.google.android.gms.ads.rewarded.RewardedAd
+import org.amshove.kluent.mock
+import org.amshove.kluent.shouldBe
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito
+import org.mockito.kotlin.doReturn
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+import org.robolectric.annotation.Config
+
+@RunWith(AndroidJUnit4::class)
+@Config(manifest = Config.NONE)
```

Comment:
Do we need this? if we can remove this, we can also remove `testImplementation "org.robolectric:robolectric..."` from `build.gradle`